# Service to put a string in the MATLAB workspace
string name       # Name to give to the string in the MATLAB workspace
string data       # String data
---
int64 ret         # Set to 1 (success) or 0 (error)
string msg        # Error description